src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -0,0 +1,86 @@
+import { loc, View, createCallout, _ } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import email from '../shared/email';
+import polling from '../shared/polling';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import { SHOW_RESEND_TIMEOUT } from '../../utils/Constants';
+
+const ResendView = View.extend(
+  {
+    className: 'hide resend-email-view',
+    events: {
+      'click a.resend-link' : 'handelResendLink'
+    },
+
+    initialize () {
+      this.add(createCallout({
+        content: `${loc('email.code.not.received', 'login')}
+        <a class='resend-link'>${loc('email.button.resend', 'login')}</a>`,
+        type: 'warning',
+      }));
+    },
+
+    handelResendLink () {
+      this.options.appState.trigger('invokeAction', this.options.resendEmailAction);
+      // Hide warning, but reinitiate to show warning again after some threshold of polling
+      if (!this.el.classList.contains('hide')) {
+        this.el.classList.add('hide');
+      }
+      this.showCalloutWithDelay();
+    },
+
+    postRender () {
+      this.showCalloutWithDelay();
+    },
+
+    showCalloutWithDelay () {
+      this.showMeTimeout = _.delay(() => {
+        this.$el.removeClass('hide');
```

Comment:
This needs to be changed to `this.el.classList.remove('hide')` which makes it inline with line 26-28.

Reply:
Shall we use more jQuery instead of native DOM API so that we dont need to worry about cross browser at all?
If our end goal was to deprecate jQuery (which definitely not short term goal), we could do it in one (or many) dedicated rounds ?

Reply:
yeah, we should pick one style for a file. Maybe avoid using jquery if possible for this entire file?

Reply:
* I think it's better to move away from jQuery & and `_` as much as possible. In the long term, we'll be able to remove the heavy dependencies (hopefully 🤞 ). Most of the functionality should be achievable using ES6. And once we have support for `optional chaining` and `nullish coalescing` we can simplify existential checks too.
* At the least, we shouldn't be mixing jQuery and JS in the same file to avoid confusion..

Reply:
- Removing jQuery could hardly be a goal because of backbone depends on it. thus I usually prefer to use jQuery so I dont have to worry my code is cross browser compatible (search for caniuse or MDN), even though there are less and less API gaps between browsers. Like I mentioned earlier, if we want to clean up jQuery, I hope it can be done by some transformation tool (e.g. if there is usage of `hasClass`, convert to `el.classList.contain`)
- Same story for underscore but it only provides utilizes hence we have better chance to avoid it.

- Mixing is always bad practice.